Xamarin.PropertyEditing.Mac/Controls/BaseDateTimeEditorControl.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using AppKit;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal abstract class BaseDateTimeEditorControl<T> : PropertyEditorControl<PropertyViewModel<T>>
+	{
+		public NSDatePicker DatePicker { get; }
+
+		public BaseDateTimeEditorControl (IHostResourceProvider hostResources)
+			: base (hostResources)
+		{
+			DatePicker = new NSDatePicker {
+				ControlSize = NSControlSize.Small,
+				DatePickerElements = NSDatePickerElementFlags.HourMinuteSecond | NSDatePickerElementFlags.YearMonthDateDay,
+				DatePickerStyle = NSDatePickerStyle.TextFieldAndStepper,
+				Font = NSFont.SystemFontOfSize (NSFont.SystemFontSizeForControlSize (NSControlSize.Small)),
+				TimeZone = Foundation.NSTimeZone.FromAbbreviation ("UTC"),
```

Comment:
Is timezone something that needs to be specified?

Reply:
yes, all times needs to be converted to UTC